src/main/java/meditracker/command/AddCommand.java

```diff
@@ -121,8 +123,8 @@ private Medication createMedication() throws NumberFormatException, NullPointerE
     private void assertionTest(MedicationManager medicationManager) {
         assert medicationManager.getTotalMedications() != 0 : "Total medications in medication " +
                 "manager should not be 0!";
-        assert DailyMedicationManager.getTotalDailyMedication() != 0 : "Total medications in daily medication " +
-                "manager should not be 0!";
+//        assert DailyMedicationManager.getTotalDailyMedication() != 0 : "Total medications in daily medication " +
```

Comment:
May i verify if these comments are needed?

Reply:
I am no longer using the getTotalDailyMedication() so I want to comment (remove) it out, otherwise the Gradle check is showing error.

Reply:
I think it is best not to check in commented out code and it would be better to remove it instead

Reply:
Alright, removed comment in c7699f5